projects/plugins/jetpack/_inc/client/newsletter/email-settings.jsx

```diff
@@ -308,7 +309,7 @@ const EmailSettings = props => {
 							),
 							{
 								settingsLink: (
-									<JetpackButton
+									<ExternalLink
 										variant="link"
 										isExternalLink={ true }
 										href={ adminUrl + 'options-general.php' }
```

Comment:
`variant` and `isExternalLink` props are not necessary, let's remove them.

This is, more or less, how it should look like:
```php
settingsLink: <ExternalLink href={ adminUrl + 'options-general.php' } />,
```